public/app/features/explore/Logs/Logs.tsx

```diff
@@ -729,7 +729,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
                 <div>
                   <InlineField label="Display results" className={styles.horizontalInlineLabel} transparent>
                     <RadioButtonGroup
-                      disabled={isFlipping}
+                      disabled={isFlipping || loading}
```

Comment:
Note: for some reason `RadioButtonGroup` is always receiving `undefined` for disabled, even if you hardcode it here to be always true. I will investigate it and open an issue or PR.